charts/evm-rollup/values.yaml

```diff
@@ -20,193 +20,171 @@ images:
+
+  # Configure the gas Limit
+  gasLimit: "50000000"
+  # If set to true the genesis block will contain extra data
+  overrideGenesisExtraData: true
+  # The hrp for bech32m addresses, unlikely to be changed
+  sequencerAddressPrefix: "astria"
+
+  ## These values are used to configure astria native bridging
+  ## Many of the fields have commented out example fields
+
+  # When using an erc20 canonical bridge, the address from which tokens will
+  # be sent via the bridge contract
+  bridgeSenderAddress: "0x0000000000000000000000000000000000000000"
+  # Configure the sequencer bridge addresses and allowed assets if using
+  # the astria canonical bridge. Recommend removing alloc values if so.
+  bridgeAddresses: []
+    # - address: "684ae50c49a434199199c9c698115391152d7b3f"
+    #   startHeight: 1
+    #   assetDenom: "nria"
+    #   assetPrecision: 9
+
+
+  ## Fee configuration
+
+  # Configure the fee collector for the evm tx fees, activated at block heights.
+  # If not configured, all tx fees will be burned.
+  feeCollectors: {}
+  # 1: "0xaC21B97d35Bf75A7dAb16f35b111a50e78A72F30"
+  # Configure EIP-1559 params, activated at block heights
+  eip1559Params: {}
+    # 1:
+    #   minBaseFee: 0
+    #   elasticityMultiplier: 2
+    #   baseFeeChangeDenominator: 8
+
+  ## Standard Eth Genesis config values
+  # An EVM chain number id, different from the astria rollup name
+  chainId: ""
+  # Configuration of Eth forks, setting to 0 will enable from height,
+  # left as is these forks will not activate.
+  cancunTime: ""
+  pragueTime: ""
+  verkleTime: ""
+  # Can configure the genesis allocs for the chain
+  alloc:
+    # Deploying the deterministic deploy proxy contract in genesis
+    # Forge and other tools use this for their CREATE2 usage, but
+    # can only be included through the genesis block after EIP-155
+    # https://github.com/Arachnid/deterministic-deployment-proxy
+    - address: "0x4e59b44847b379578588920cA78FbF26c0B4956C"
+      value:
+        balance: "0"
+        code: "0x7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe03601600081602082378035828234f58015156039578182fd5b8082525050506014600cf3"
+    # Example of simple genesis account funding
+    # - address: "0xaC21B97d35Bf75A7dAb16f35b111a50e78A72F30"
+    #   value:
+    #   # The balance must be a string due to size
+    #   balance: "1000000000000000000000000000"
 
 config:
   # The level at which core astria components will log out
   # Options are: error, warn, info, and debug
   logLevel: "debug"
 
-  rollup:
-    # string name for the rollup chain, used to generate Rollup ID
-    name: ""
-    # EVM network ID used by the chain
-    networkId: ""
-    # Can be set to 'pebble' or 'leveldb'
-    dbEngine: pebble
+  geth:
     # Set to true to keep history of all blocks
     archiveNode: false
     # Set to true to clear the mempool on startup/restart
     purgeMempool: false
+    flags:
```

Comment:
this allows any of the flags on geth to be set when running the chart, for them to have a value or not, and to set a `condition` which can be executed in an helm `if` statement for whether or not to set the flag.